tests/testthat/test-assertions.R

```diff
@@ -320,22 +320,33 @@ test_that("assert_character_scalar Test 17: error if `arg` not in values", {
 })
 
 ## Test 18: error if `arg` not a character vector ----
-test_that("assert_character_scalar Test 18: error if `arg` not a character vector", {
+test_that("assert_character_vector Test 18: error if `arg` not a character vector", {
   arg <- c(1, 2, 3)
 
   expect_error(
+    assert_character_vector(arg),
+    class = "assert_character_vector"
+  )
+
+  expect_snapshot(
+    error = TRUE,
     assert_character_vector(arg)
   )
 })
 
 ## Test 19: error if `arg` is not in values ----
-test_that("assert_character_scalar Test 19: error if `arg` is not in values", {
+test_that("assert_character_vector Test 19: error if `arg` is not in values", {
   example_fun <- function(character) {
     assert_character_vector(character, values = c("test", "oak"))
   }
 
```

Comment:
Is it worth testing with `named = TRUE` to show that argument name from `assert_character_vector` is displayed, even though the message is coming from `assert_named` ?

Reply:
i think that is a great test @millerg23 ! but some followup questions about adding a new test.
- the unit testing file is grouped by function
- every unit test is numbered

if I were to add a new unit test for an assert function, is the expectation that I renumber every subsequent test in the file? (there are almost 100 tests)
@bms63 @bundfussr @manciniedoardo

Reply:
> if I were to add a new unit test for an assert function, is the expectation that I renumber every subsequent test in the file? (there are almost 100 tests)

Yes, but you shouldn't do it manually. We have an addin for this. See https://pharmaverse.github.io/admiraldev/dev/articles/unit_test_guidance.html#addin-admiraldevformat_test_that_file

Reply:
ahh, thanks @bundfussr ✅

Reply:
test added!

Reply:
@ddsjoberg the snapshot needs updated, as test numbers changed! Have 21 and 50 instead of 22 and 51 in snapshot!